frontend/src/metabase/actions/containers/ActionParametersInputModal.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import Modal from "metabase/components/Modal";
+import ModalContent from "metabase/components/ModalContent";
+
+interface Props {
+  onClose: () => void;
+  title: string;
+  children: React.ReactElement;
+}
+
+export default function ActionParametersInputModal({
+  onClose,
+  title,
+  children,
+}: Props) {
+  return (
+    <Modal onClose={onClose}>
+      <ModalContent title={title} onClose={onClose}>
+        {children}
+      </ModalContent>
+    </Modal>
+  );
+}
```

Comment:
Can we either remove this component or replace `children` with `ActionParametersInputForm`?